src/App.jsx

```diff
@@ -14,33 +16,78 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+function getSorted(obj, way) {
```

Comment:
This is a code quality issue. Method name should start with a verb. It is a good practice to name functions or methods as verbs because they represent actions. You could rename this function to `sortGoods` or `getSortedGoods`.